sources/packages/web/src/router/StudentRoutes.ts

```diff
@@ -25,6 +26,15 @@ export const studentRoutes: Array<RouteRecordRaw> = [
         path: "login",
         name: StudentRoutesConst.LOGIN,
         component: Login,
+        beforeEnter: (to, from, next) => {
+          if (!AppConfigService.shared.authService?.authenticated) {
```

Comment:
It could be ok for now, but what I have been mentioning is that we should move to a model like below.
![image](https://user-images.githubusercontent.com/61259237/110564600-f2991080-8101-11eb-9a34-644f2cd81acc.png)

Reply:
This is going beyond login page, directly to KeyCloak? Do we really need that?